pandas/core/ops.py

```diff
@@ -1298,18 +1371,19 @@ def f(self, other, axis=0):
                            comp_method=_comp_method_PANEL,
                            bool_method=_arith_method_PANEL)
 
+panel_flex_funcs = dict(flex_arith_method=_flex_method_PANEL,
```

Comment:
are you adding support for this?

Reply:
This changes zero logic, just puts a dict here instead of passing these directly in panel.